clients/firebaseInteractor.ts

```diff
@@ -0,0 +1,32 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/storage";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDV3FObRh1ylPJkgowm1eXQw4XIykKkbDQ",
+  authDomain: "mgh-video-journal.firebaseapp.com",
+  projectId: "mgh-video-journal",
+  storageBucket: "mgh-video-journal.appspot.com",
+  messagingSenderId: "253050399833",
+  appId: "1:253050399833:web:093bc484f3816d35fd9548",
+  measurementId: "G-TVJY0CJ7CV",
+};
+
+if (firebase.apps.length == 0) {
+  firebase.initializeApp(firebaseConfig);
+}
+
+export const signIn = async (email: string): Promise<void> => {
+  await firebase.auth().signInAnonymously();
+};
+
+export const uploadFileToFirebase = async (
+  name: string,
+  fileURI: string
+): Promise<void> => {
+  let file = await fetch(fileURI);
```

Comment:
fetch is great, but is it supported on all (most) (enough?) mobile devices? idk if you guys have already checked, but if not we'll have to use a polyfill

Reply:
We tested on iPhone/iPad and it seems to work. I can investigate to see if it doesn't work on any likely devices.

Reply:
i just did some research into it and it turns out that because we're writing an app and not a browser, we don't actually have to worry about supporting browsers. react native actually supplies their own fetch api to simulate being in a browser, so it feels like we are for web devs, even tho we aren't (lol).

Reply:
https://github.com/react-native-community/fetch

https://github.com/facebook/react-native/blob/master/Libraries/Network/fetch.js

if your'e curious

Reply:
awesome, thanks for checking!